api/swagger-spec/v1.json

```diff
@@ -19005,6 +19005,10 @@
        "$ref": "v1.HostAlias"
       },
       "description": "HostAliases is an optional list of hosts and IPs that will be injected into the pod's hosts file if specified. This is only valid for non-hostNetwork pods."
+     },
+     "priorityName": {
+      "type": "string",
+      "description": "If specified, indicates the pod's priority. `system` is a special keyword which indicates the highest priority. Any other name must be defined in Admission Controller config or pod will be rejected. If not specified, the pod priority will be zero."
```

Comment:
I will update all the generated files eventually. For some reason, it is not that easy to update them. If I run hack/update-all.sh it does not work and corrupts my files. I have to run individual update scripts in a not-so-well-known order to make it work.